src/Arcus.Testing.Integration.DataFactory/TemporaryDataFlowDebugSession.cs

```diff
@@ -1,16 +1,17 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.Linq;
-using System.Threading.Tasks;
-using Azure;
+﻿using Azure;
 using Azure.Core;
 using Azure.Identity;
 using Azure.ResourceManager;
 using Azure.ResourceManager.DataFactory;
 using Azure.ResourceManager.DataFactory.Models;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
```

Comment:
Can revert these changes and place the `using System.*;` back on top?
I think that there might be a 'order on save' option activated in your IDE which does this and which I would recommonend to diactivate on open-source code, as you would ruin every file, each time.

In this case, placing the 'general, built-in `System` ones' first makes the usings more clear as the specific ones will be grouped together.